documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-high-volume-config-properties-{context}']
+= Handling high volumes of messages
+
+[role="_abstract"]
+If your Strimzi deployment needs to handle a high volume of messages, you can use configuration options to optimize for throughput and latency. 
+
+Producer and consumer configuration can help control the size and frequency of requests to Kafka brokers. 
+For more information on the configuration options, see the following:
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the producers and consumers used by Kafka Connect runtime and MirrorMaker 2.0 source connectors and sink connectors.
+
+Source connectors:: 
+* Consumers retrieve messages from a source Kafka cluster for MirrorMaker 2.0. As Kafka Connect fetches data from an external data system, consumers are not usually required for source connectors.  
+* Producers send messages to a target Kafka cluster for Kafka Connect and MirrorMaker 2.0.
+
+Sink connectors:: 
+* Consumers retrieve data from a source Kafka cluster for Kafka Connect.
+* Producers push data to an external data system for Kafka Connect.
```

Comment:
We should remove this. There are typically no producers in sink connectors. The target system is not Kafka so the connector use the system's own protocol to send messages.